core/templates/dev/head/galleries/Gallery.js

```diff
@@ -203,6 +203,51 @@ oppia.controller('Gallery', [
       $rootScope.loadingMessage = '';
     });
 
+    $scope.leftCheck = function(ind) {
+      var tilesHtml = '.oppia-gallery-tiles:eq(n)'.replace('n', ind);
+
+      // Checks if it is at the left most point.
+      return parseInt($(tilesHtml).css('left')) >= 0;
+    };
+
+    $scope.scrollLeft = function(ind) {
+      if (!$scope.leftCheck(ind)) {
+        var tilesHtml = '.oppia-gallery-tiles:eq(n)'.replace('n', ind);
+        $(tilesHtml).animate({
+          left: '+=211.953'
+        }, 500);
+      }
+    };
+
+    $scope.rightLowerLimits = [];
+    $scope.carouselDisplayWidth = $('.oppia-gallery-tiles-carousel').width();
+    $scope.rightCheck = function(ind, galleryGroup) {
```

Comment:
Can this (and leftCheck) start with a verb, so that it's clear that it's a function? It's also not clear what exactly is being checked, so I think it needs a clearer name (and possibly a docstring, but hopefully you find a clear name that removes the need for the latter).